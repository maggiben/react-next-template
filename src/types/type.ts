

export type Person = {
  id: string;
  name: string;
  age: number,
  identification: {
    type: string,
    number: number;
  },
  profession: string;
  faceapi: boolean;
  email: {
    address: string;
    confirmed: boolean;
  }
  city: string;
  cp: string;  
  selected: boolean;
  status: {
    label: string;
    color: string;
  }
}
