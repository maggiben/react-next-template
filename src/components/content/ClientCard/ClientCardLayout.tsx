import styled from "styled-components";
import { 
  Grid, 
  GridItem, 
  Heading, 
  Label,
  Button,
} from '@fravega-it/bumeran-ds-fvg'
import { useTranslation } from 'react-i18next';
import { Person } from 'types/type';
import { useMediaQuery } from 'usehooks-ts';
import DataContainer from '@components/DataContainer/DataContainer';
import ValidationList from '../ValidationList/ValidationList';
import { SpaceRight, SpaceTop } from '@components/Spacing/Spacing';

const Card = styled.div`
  position: relative;
  display: block;
  width: 100%;
  border-width: 1px;
  border-style: solid;
  padding: ${({ theme }) => theme.spacing.m};
  background-color: ${({ theme }) => theme.colors.white};
  border-color: ${({ theme }) => theme.colors.neutral[300]};
  border-radius: ${({ theme }) => theme.borderRadius.m};
`;

const Centered = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
`;

type ClientCardLayoutProps = {
  person: Person;
}

const ClientCardLayout = (props: ClientCardLayoutProps) => {
  const { t } = useTranslation();
  const { person } = props;
  const columns = useMediaQuery('(min-width: 768px)') ? 3 : 1;

  const clientData: Record<string, string>[] = [
    {[t('birth date')]: new Date(person.age).toLocaleDateString()}, 
    {[t('email')]: person.email.address}, 
    {[t('address')]: `${person.address.street} • ${person.address.number} • ${person.address.city}`}, 
    {[t('phone')]: `${person.phone.codeArea.toString()} - ${person.phone.number.toString()}`},
    {[t(person.identification.type)]: person.identification.number.toString()}, 
    {[t('id expiration date')]: new Date(person.identification.expiration).toLocaleDateString()}, 
  ];

  /* TODO: quick fix make batter */
  const colors = {
    onboardingfull: true,
    onboardingincompleto: false,
    dnicaduco: false,
  };

  const labels = {
    onboardingfull: t('onboarding full'),
    onboardingincompleto: t('onboarding incomplete'),
    dnicaduco: t('dni expired'),
  };

  return (
    <Card>
      <Grid>
        <GridItem xs={6} justifySelf="start" alignSelf="center">
          <Centered>
            <Heading size="s">{person.name} {person.lastname}</Heading><SpaceRight size="s" /><Label label={labels[person.status]} color={colors[person.status] ? 'green' : 'red'}/>
          </Centered>
        </GridItem>
        <GridItem xs={6} alignSelf="center" justifySelf="end">
          <Button label={t('resend email')} variant="primary" size="s" onClick={() => alert('hola')}/>
        </GridItem>
        <GridItem xs={12}>
          <ValidationList person={person} />
          <SpaceTop size="s" />
          <DataContainer data={clientData} columns={columns} />
        </GridItem>
      </Grid>
    </Card>
  )
};

  export default ClientCardLayout;
