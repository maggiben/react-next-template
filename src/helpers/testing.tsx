import { default as baseRenderer } from 'react-test-renderer';
import { render, RenderResult } from '@testing-library/react';
import i18n from '@utils/i18n';
import { I18nextProvider } from 'react-i18next';
import { RecoilRoot } from 'recoil';
import { ThemeProvider } from 'styled-components';
import { theme, IDefaultTheme } from '@fravega-it/bumeran-ds-fvg';
import React, { ReactElement } from 'react';
import '@testing-library/jest-dom'

/**
 * Helper used to render components for testing.
 * The function appends the theme to the children.
 * */
interface IRenderer {
  create: (children: ReactElement) => RenderResult;
  baseRenderer: (children: ReactElement) => baseRenderer.ReactTestRenderer;
  renderWithTheme: (children: ReactElement, theme: IDefaultTheme) => RenderResult;
  renderWithI18n: (children: ReactElement) => RenderResult;
}

export const renderer: IRenderer = {
  create: (children: ReactElement): RenderResult =>
    render(
      <ThemeProvider theme={theme}>{children}</ThemeProvider>
  ),
  renderWithTheme: (children: ReactElement, theme: IDefaultTheme): RenderResult =>
    render(
      <ThemeProvider theme={theme}>{children}</ThemeProvider>
  ),
  baseRenderer: (children: ReactElement): baseRenderer.ReactTestRenderer =>
    baseRenderer.create(
      <ThemeProvider theme={theme}>{children}</ThemeProvider>
  ),
  renderWithI18n: (children: ReactElement): RenderResult => 
    render(
      <ThemeProvider theme={theme}>
        <I18nextProvider i18n={i18n}>{children}</I18nextProvider>
      </ThemeProvider>
  ),
};