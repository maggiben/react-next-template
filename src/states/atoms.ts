import { atom } from 'recoil';

export const personState = atom({
  key: 'personState',
  default: {
    id: '',
    name: '',
    lastname: '',
    age: 0,
    identification: {
      type: '',
      number: 0,
      expiration: '',
    },
    faceapi: false,
    email: {
      address: '',
      confirmed: false,
    },
    renaper: {
      confirmed: false,
      expiration: '',
    },
    profession: '',
    city: '',
    cp: '',
    selected: false,
    status: {
      label: '',
      color: '',
    },
  },
});

export const currentFilterState = atom({
  key: 'currentFilterState',
  default: {
    name: undefined, 
    email: undefined, 
    cuid: undefined,
    documentType: undefined,
    documentNumber: undefined,
  },
});
