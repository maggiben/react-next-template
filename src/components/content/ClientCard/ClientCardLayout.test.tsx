import React from 'react';
import { renderer } from '@helpers/testing';
import { act, RenderResult } from '@testing-library/react';
import 'jest-styled-components';
import { useRouter } from 'next/router';
import { useFetch, useMediaQuery } from 'usehooks-ts';
import { RecoilRoot } from 'recoil';
import '@testing-library/jest-dom';
import {
  screen,
  getAllByRole,
} from "@testing-library/react";
import { personState, personsState } from '@states/atoms';
import { expect } from '@jest/globals';
import ClientCardLayout from './ClientCardLayout';
import { Person } from 'types/type';

const clients: Person[] = [
  {
      "id": "001",
      "name": "Federico",
      "lastname": "Garcia",
      "age": "1983-01-24",
      "validations": {
          "facephi": true,
          "renaper": true,
          "email": true,
          "phone": true,
          "identificationExpired": true
      },
      "identification": {
          "type": "dni",
          "number": 30000000,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "email": {
          "address": "federico@gmail.com",
          "confirmed": false
      },
      "selected": false,
      "phone": {
          "number": 55512345,
          "prefix": 11,
          "codeArea": 356
      },
      "status": "onboardingfull",
      "address": {
          "zipCode": "1100",
          "street": "AV. Corrientes",
          "number": "1700",
          "floor": "0",
          "apartment": "C",
          "city": "Neuquen",
          "state": "Neuquen",
          "country": "Arg"
      }
  },
  {
      "id": "002",
      "name": "Facundo",
      "lastname": "Perez",
      "age": "2001-03-02",
      "validations": {
          "facephi": false,
          "renaper": true,
          "email": true,
          "phone": false,
          "identificationExpired": false
      },
      "identification": {
          "type": "dni",
          "number": 30000001,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "email": {
          "address": "facundo@gmail.com",
          "confirmed": false
      },
      "selected": false,
      "phone": {
          "number": 55512345,
          "prefix": 11,
          "codeArea": 299
      },
      "status": "dnicaduco",
      "renaper": {
          "confirmed": true,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "address": {
          "zipCode": "1414",
          "street": "Castillo",
          "number": "1414",
          "floor": "1",
          "apartment": "A",
          "city": "CABA",
          "state": "CABA",
          "country": "Arg"
      }
  },
  {
      "id": "003",
      "name": "Facundo",
      "lastname": "Perez",
      "age": "1961-02-24",
      "validations": {
          "facephi": true,
          "renaper": false,
          "email": false,
          "phone": false,
          "identificationExpired": true
      },
      "identification": {
          "type": "dni",
          "number": 30000001,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "email": {
          "address": "facundo@gmail.com",
          "confirmed": false
      },
      "selected": false,
      "phone": {
          "number": 1234567,
          "prefix": 11,
          "codeArea": 264
      },
      "status": "onboardingfull",
      "renaper": {
          "confirmed": true,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "address": {
          "zipCode": "8400",
          "street": "Mitre",
          "number": "12",
          "floor": "1",
          "apartment": "A",
          "city": "S.C. de Bariloche",
          "state": "Rio Negro",
          "country": "Arg"
      }
  },
  {
      "id": "004",
      "name": "Jeremías",
      "lastname": "Springfield",
      "age": "2003-06-05",
      "validations": {
          "facephi": true,
          "renaper": false,
          "email": false,
          "phone": true,
          "identificationExpired": false
      },
      "identification": {
          "type": "dni",
          "number": 30000002,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "email": {
          "address": "jeremías@gmail.com",
          "confirmed": true
      },
      "selected": false,
      "phone": {
          "number": 5559876,
          "prefix": 22,
          "codeArea": 987
      },
      "status": "onboardingincompleto",
      "renaper": {
          "confirmed": true,
          "expiration": "2023-12-29T12:33:32.740Z"
      },
      "address": {
          "zipCode": "4400",
          "street": "Palacios",
          "number": "12",
          "floor": "7",
          "apartment": "A",
          "city": "Salta",
          "state": "Salta",
          "country": "Arg"
      }
  }
];

const person: Person = {
  "id": "001",
  "name": "Federico",
  "lastname": "Garcia",
  "age": "1983-01-24",
  "validations": {
      "facephi": true,
      "renaper": true,
      "email": true,
      "phone": true,
      "identificationExpired": true
  },
  "identification": {
      "type": "dni",
      "number": 30000000,
      "expiration": "2023-12-29T12:33:32.740Z"
  },
  "email": {
      "address": "federico@gmail.com",
      "confirmed": false
  },
  "selected": false,
  "phone": {
      "number": 55512345,
      "prefix": 11,
      "codeArea": 356
  },
  "status": "onboardingfull",
  "address": {
      "zipCode": "1100",
      "street": "AV. Corrientes",
      "number": "1700",
      "floor": "0",
      "apartment": "C",
      "city": "Neuquen",
      "state": "Neuquen",
      "country": "Arg"
  }
};

jest.mock('usehooks-ts', () => ({
  useRouter: jest.fn(),
  useFetch: jest.fn(),
  useMediaQuery: jest.fn(),
}));
const mockUseMediaQuery = useMediaQuery as jest.Mock;

describe('ClientCard', () => {
  beforeEach(() => {

  });

  it('renders the ClientCard component', async () => {
    const tree = renderer.create(
      <ClientCardLayout person={person} />
    );
    expect((tree as RenderResult).baseElement).toMatchSnapshot();
  });

  it('renders the ClientCard component', async () => {
    const tree = renderer.create(
      <ClientCardLayout person={{ ...person, validations: { email: false }}} />
    );
    expect((tree as RenderResult).baseElement).toMatchSnapshot();
  });
});
