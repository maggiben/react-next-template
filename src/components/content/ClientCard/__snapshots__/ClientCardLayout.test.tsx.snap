// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientCard renders the ClientCard component 1`] = `"<div><div class="ClientCardLayout__Card-sc-opbt8b-0 iSKXzI"><div class="sc-jsJBEP gDratY"><div class="sc-eeDRCY qRuhs"><div class="ClientCardLayout__Centered-sc-opbt8b-1 gEMTqb"><div role="heading" aria-level="5" class="sc-kAyceB eTjleT">Federico Garcia</div><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><div color="green" class="sc-bmzYkS evAJPt"><svg color="green" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm fxbHNO"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.996 9.56a.778.778 0 0 1 0 1.1l-7.78 7.78a.778.778 0 0 1-1.1 0l-3.112-3.112a.778.778 0 0 1 1.1-1.1l2.562 2.562 7.23-7.23a.778.778 0 0 1 1.1 0Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Onboarding completo</span></div></div></div><div class="sc-eeDRCY fbRmQB"><div class="ValidationList__Card-sc-zx6xp2-0 edAsZc"><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="green" class="sc-bmzYkS evAJPt"><svg color="green" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm fxbHNO"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.996 9.56a.778.778 0 0 1 0 1.1l-7.78 7.78a.778.778 0 0 1-1.1 0l-3.112-3.112a.778.778 0 0 1 1.1-1.1l2.562 2.562 7.23-7.23a.778.778 0 0 1 1.1 0Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Email Validado</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente ya validó su correo electrónico</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="green" class="sc-bmzYkS evAJPt"><svg color="green" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm fxbHNO"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.996 9.56a.778.778 0 0 1 0 1.1l-7.78 7.78a.778.778 0 0 1-1.1 0l-3.112-3.112a.778.778 0 0 1 1.1-1.1l2.562 2.562 7.23-7.23a.778.778 0 0 1 1.1 0Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Validado en Renaper</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente ya validó su identidad en el Registro Nacional de las Personas</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="green" class="sc-bmzYkS evAJPt"><svg color="green" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm fxbHNO"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.996 9.56a.778.778 0 0 1 0 1.1l-7.78 7.78a.778.778 0 0 1-1.1 0l-3.112-3.112a.778.778 0 0 1 1.1-1.1l2.562 2.562 7.23-7.23a.778.778 0 0 1 1.1 0Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Validado en Facephi</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente ya validó su identidad biométrica</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 fcjMYq"></div><table class="DataContainer__Table-sc-122pn0e-0 iBvYho"><tbody><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Fecha de Nacimiento</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">1/23/1983</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Email</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">federico@gmail.com</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Dirección Legal</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">AV. Corrientes • 1700 • Neuquen</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Teléfono</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">356 - 55512345</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">DNI</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">30000000</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Fecha de caducidad del documento</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">12/29/2023</span></div></td></tr></tbody></table></div></div></div></div>"`;

exports[`ClientCard renders the ClientCard component 2`] = `"<div><div class="ClientCardLayout__Card-sc-opbt8b-0 iSKXzI"><div class="sc-jsJBEP gDratY"><div class="sc-eeDRCY qRuhs"><div class="ClientCardLayout__Centered-sc-opbt8b-1 gEMTqb"><div role="heading" aria-level="5" class="sc-kAyceB eTjleT">Federico Garcia</div><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><div color="green" class="sc-bmzYkS evAJPt"><svg color="green" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm fxbHNO"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.996 9.56a.778.778 0 0 1 0 1.1l-7.78 7.78a.778.778 0 0 1-1.1 0l-3.112-3.112a.778.778 0 0 1 1.1-1.1l2.562 2.562 7.23-7.23a.778.778 0 0 1 1.1 0Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Onboarding completo</span></div></div></div><div class="sc-eeDRCY juJzqb"><button class="sc-fHjqPf dUMcZa"><label aria-label="Reenviar email"><div color="inherit" role="heading" aria-level="6" class="sc-kAyceB ffHZfz">Reenviar email</div></label><svg color="white" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm jokLXb"><path fill-rule="evenodd" clip-rule="evenodd" d="M6.22 10.11a2.334 2.334 0 0 1 2.334-2.334h10.892a2.334 2.334 0 0 1 2.334 2.334v7.779a2.334 2.334 0 0 1-2.334 2.335H8.554A2.334 2.334 0 0 1 6.22 17.89V10.11Zm1.556 2.232v5.548c0 .43.349.778.778.778h10.892c.43 0 .778-.348.778-.778v-5.548l-4.93 3.286a2.334 2.334 0 0 1-2.589 0l-4.929-3.286Zm12.448-1.87-5.793 3.861a.778.778 0 0 1-.862 0l-5.793-3.861v-.362c0-.43.349-.778.778-.778h10.892c.43 0 .778.348.778.778v.362Z"></path></svg></button></div><div class="sc-eeDRCY fbRmQB"><div class="ValidationList__Card-sc-zx6xp2-0 edAsZc"><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="red" class="sc-bmzYkS hwfFMR"><svg color="red" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm dFcenU"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.98 8.168c.898-1.555 3.142-1.555 4.04 0l5.387 9.331c.898 1.555-.224 3.5-2.02 3.5H8.613c-1.796 0-2.918-1.945-2.02-3.5l5.387-9.33Zm2.693.778a.778.778 0 0 0-1.346 0l-5.388 9.33a.778.778 0 0 0 .674 1.167h10.774a.778.778 0 0 0 .674-1.166l-5.388-9.331ZM14 11.667c.43 0 .778.348.778.778V14a.778.778 0 0 1-1.556 0v-1.555c0-.43.349-.778.778-.778Zm-.778 5.443c0-.43.349-.777.778-.777h.008a.778.778 0 0 1 0 1.555H14a.778.778 0 0 1-.778-.778Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">Email no validado</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente aún no validó su correo electrónico</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="red" class="sc-bmzYkS hwfFMR"><svg color="red" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm dFcenU"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.98 8.168c.898-1.555 3.142-1.555 4.04 0l5.387 9.331c.898 1.555-.224 3.5-2.02 3.5H8.613c-1.796 0-2.918-1.945-2.02-3.5l5.387-9.33Zm2.693.778a.778.778 0 0 0-1.346 0l-5.388 9.33a.778.778 0 0 0 .674 1.167h10.774a.778.778 0 0 0 .674-1.166l-5.388-9.331ZM14 11.667c.43 0 .778.348.778.778V14a.778.778 0 0 1-1.556 0v-1.555c0-.43.349-.778.778-.778Zm-.778 5.443c0-.43.349-.777.778-.777h.008a.778.778 0 0 1 0 1.555H14a.778.778 0 0 1-.778-.778Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">No Validado en Renaper</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente no validó su identidad en el Registro Nacional de las Personas</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div><div><div spacing="s" class="sc-jsJBEP eNXzoM"><div class="sc-eeDRCY gNxwjx"><div class="ValidationList__Centered-sc-zx6xp2-1 fOVA-Do"><div color="red" class="sc-bmzYkS hwfFMR"><svg color="red" width="20px" fill="none" viewBox="0 0 28 28" class="sc-kpDqfm dFcenU"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.98 8.168c.898-1.555 3.142-1.555 4.04 0l5.387 9.331c.898 1.555-.224 3.5-2.02 3.5H8.613c-1.796 0-2.918-1.945-2.02-3.5l5.387-9.33Zm2.693.778a.778.778 0 0 0-1.346 0l-5.388 9.33a.778.778 0 0 0 .674 1.167h10.774a.778.778 0 0 0 .674-1.166l-5.388-9.331ZM14 11.667c.43 0 .778.348.778.778V14a.778.778 0 0 1-1.556 0v-1.555c0-.43.349-.778.778-.778Zm-.778 5.443c0-.43.349-.777.778-.777h.008a.778.778 0 0 1 0 1.555H14a.778.778 0 0 1-.778-.778Z"></path></svg><span color="inherit" class="sc-iGgWBj djeqlO">No Validado en Facephi</span></div><div class="Spacing__SpaceRight-sc-11df7di-0 bsMTuc"></div><span color="neutral" class="sc-iGgWBj emVxnH">El cliente no validó su identidad biométrica</span></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 gIigKZ"></div></div></div><div class="Spacing__SpaceTop-sc-11df7di-1 fcjMYq"></div><table class="DataContainer__Table-sc-122pn0e-0 iBvYho"><tbody><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Fecha de Nacimiento</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">1/23/1983</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Email</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">federico@gmail.com</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Dirección Legal</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">AV. Corrientes • 1700 • Neuquen</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Teléfono</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">356 - 55512345</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">DNI</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">30000000</span></div></td></tr><tr><td><div class="DataContainer__Centered-sc-122pn0e-1 byuuFh"><span color="neutral" class="sc-iGgWBj emVxnH">Fecha de caducidad del documento</span><div class="Spacing__SpaceRight-sc-11df7di-0 latkgH"></div><span class="sc-iGgWBj GArsk">12/29/2023</span></div></td></tr></tbody></table></div></div></div></div>"`;
